scripts/eval_hyde_expand1.py

```
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+"""HyDE + 1 expansion"""
+import sys
+sys.path.insert(0, '/app')
+exec(open('/app/scripts/eval_hyde_only.py').read().replace(
```

Ruff (0.14.5)

Use of `exec` detected

(S102)